src/lib/containers/access_requirement_list/AccessRequirementList.tsx

```diff
@@ -42,10 +42,11 @@ type AccessRequirementAndStatus = {
 }
 
 export type AccessRequirementListProps = {
-  entityId: string
+  entityId: string  // will show this entity info
   accessRequirementFromProps?: Array<AccessRequirement>
   onHide?: Function
   renderAsModal?: boolean
+  numberOfFilesEffected?: number // if provided, will show this instead of the entity information
```

Comment:
From the Actions Required context, the design calls for showing the number of files effected by the unmet AR instead of a specific entity.  For example, "12 Files" instead of "file1.bam", where the 12 files were files that the user added to their Download List v2.  For ManagedACTAccessRequirements, this has the potential to be a little clunky for the user, since the email notifications will point to a specific file (and we select essentially a random entity that is covered by this AR).  If the user goes through the process, they will be granted access to the target files, so I'm not suggesting for us to change the ManagedACTAccessRequirement services at this time.

Reply:
I'm unfamiliar with the ManagedACTAccessRequirements component, but this behavior makes sense to me in the context of the download list.

Reply:
Nit: I think effected -> affected

Reply:
Right you are!